airflow-core/docs/security/secrets/secrets-backend/index.rst

```diff
@@ -64,12 +66,21 @@ The ``[secrets]`` section has the following options:
     [secrets]
     backend =
     backend_kwargs =
+    backends_order =
 
 Set ``backend`` to the fully qualified class name of the backend you want to enable.
 
 You can provide ``backend_kwargs`` with json and it will be passed as kwargs to the ``__init__`` method of
 your secrets backend.
 
+``backends_order`` comma-separated list of secret backends. These backends will be used in the order they are specified.
+Please note that the ``environment_variable`` and ``metastore`` are required values and cannot be removed
```

Comment:
If not specified, do we plan to auto add them? And in which order?

Reply:
Hello @uranusjr  as it listed in the code here: https://github.com/apache/airflow/pull/45931/files#diff-12a2a29eab84b723e9342e89877c875d3e979220a53884a9103d8c067ffdc286R2409

Exception will be raised in this case.